notebooks/19.Feature_Selection.ipynb

```diff
@@ -11,9 +11,9 @@
    },
    "outputs": [],
    "source": [
+    "%matplotlib inline\n",
```

Comment:
That doesn't matter, does it?

Reply:
No, I do not believe it does, mostly a pet peeve of mine, which I changed in my own version.